src/main/java/Person.java

```diff
@@ -85,34 +99,39 @@ public void setFather(Person father) {
         this.father = father;
     }
 
-    public void setSiblings(List<Person> siblings) {
-        this.siblings = siblings;
-    }
 
     public void setPets(List<Pet> pets) {
         this.pets = pets;
     }
 
     public void setChildren(List<Person> children) {
-        this.children = children;
+        if(children == null) {
+            throw new IllegalArgumentException("children cannot be null");
```

Comment:
Exceptions hebben we op dit moment in de cursus nog niet behandeld. Dit moeten we even bespreken wat we hiermee doen.

Reply:
herschreven